### PR TITLE
Feature/demo statictableview

### PR DESCRIPTION
**Pull request by andykkt**

### CODE REVIEW

File: BFWControls Demo/Modules/Table/Controller/ExcludeCellDemoViewController.swift [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        removeCell(indexPath: indexPath)
+    }
+
+    fileprivate func removeCell(indexPath: IndexPath) {
+        if let cell = tableView.cellForRow(at: indexPath) {
+            tableView.beginUpdates()
+            selectedCells.append(cell)
+            tableView.deleteRows(at: [indexPath], with: .automatic)
+            tableView.endUpdates()
```
Comment (barefeettom) [inline BFWControls Demo/Modules/Table/Controller/ExcludeCellDemoViewController.swift]: Is `beginUpdates()` and `endUpdates()` needed here? Doesn't the single `deleteRows()` do its own animation?

File: BFWControls Demo/Modules/Transition/Controller/Transition.storyboard [hunk 2]
```diff
@@ -73,10 +73,10 @@
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
                             <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="Sry-37-0Xp">
-                                <rect key="frame" x="113" y="318" width="150" height="30"/>
+                                <rect key="frame" x="112.5" y="318.5" width="150" height="30"/>
                                 <state key="normal" title="Back to previous view"/>
                                 <connections>
-                                    <segue destination="VKL-TU-a5K" kind="unwind" unwindAction="unwindToTransitionWithSegue:" id="Mj6-Ss-0v9"/>
+                                    <segue destination="VKL-TU-a5K" kind="unwind" identifier="backSegue" unwindAction="unwindToTransitionWithSegue:" id="Mj6-Ss-0v9"/>
```
Comment (barefeettom) [inline BFWControls Demo/Modules/Transition/Controller/Transition.storyboard]: Please remove `Segue` from the names of segues.

**Review by barefeettom — CHANGES_REQUESTED** [review]

File: BFWControls Demo/Modules/Transition/Controller/Transition.storyboard (hunk 2 above)
Comment (andykkt) [reply BFWControls Demo/Modules/Transition/Controller/Transition.storyboard]: Removed `Segue` from segue names

File: BFWControls Demo/Modules/Table/Controller/ExcludeCellDemoViewController.swift (hunk 1 above)
Comment (andykkt) [reply BFWControls Demo/Modules/Table/Controller/ExcludeCellDemoViewController.swift]: It is not required hence removed

**Pull request merged**